### PR TITLE
readme - bring on par with manage dependencies feature

### PR DESCRIPTION
**Pull request by martenson**
was 404

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/
 Not all dependencies are included for the tools provided in the sample
 ``tool_conf.xml``. A full list of external dependencies is available at:
 
-https://wiki.galaxyproject.org/Admin/Tools/ToolDependencies
+https://galaxyproject.org/admin/config/tool-dependencies/
```
Comment (nsoranzo) [inline README.rst]: That's a different page, it should be the equivalent of https://moin.galaxyproject.org/Admin/Tools/ToolDependencies

Comment (martenson) [reply README.rst]: Good point. That one seems lost for now though (https://github.com/galaxyproject/galaxy-hub/blob/master/src/admin/tools/ToolDependencies.md)

Comment (nsoranzo) [reply README.rst]: Or probably better rephrase the paragraph above mentioning that dependencies for the pre-installed tools can be installed through the admin interface under "Manage dependencies".

Comment (martenson) [reply README.rst]: What about just killing the whole paragraph?

edit: or that^, yeah I will do that

File: README.rst [hunk 2]
```diff
@@ -44,17 +44,17 @@ browser at:
...
-``tool_conf.xml``. A full list of external dependencies is available at:
+You may wish to make changes from the default configuration. This can be
+done in the ``config/galaxy.ini`` file.
 
-https://galaxyproject.org/admin/config/tool-dependencies/
+Tools can be either installed from the Tool Shed or added manually.
+ For details please see the `tutorial <https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/>`__.
+Note that not all dependencies for the tools provided in the
+``tool_conf.xml.sample`` are included. To install them please visit
+"Manage dependencies" in the admin interface.
```
Comment (nsoranzo) [inline README.rst:57]: It's actually called "Manage Tool Dependencies", sorry for the previous misleading suggestion.

File: README.rst [hunk 3]
```diff
@@ -64,4 +64,4 @@ Please see `CONTRIBUTING.md <CONTRIBUTING.md>`_ .
 Roadmap
 =============================
 
-Interested in the next steps for Galaxy? Take a look here: https://github.com/galaxyproject/galaxy/issues/1928
+Interested in the next steps for Galaxy? Take a look at the `roadmap <https://github.com/galaxyproject/galaxy/issues/1928>`__
```
Comment (nsoranzo) [inline README.rst]: Add period at the end.

File: README.rst (hunk 2 above)
Comment (martenson) [reply README.rst:57]: The menu item is `Manage dependencies` (on dev)

Comment (nsoranzo) [reply README.rst:57]: True, it has been changed in `dev`, it's still "Manage Tool Dependencies" in 17.09. Not sure what is best here, but I suppose it's fine as it is now.

**Review by nsoranzo — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (martenson) [reply README.rst:57]: Given our speed of updates in this readme we are fine looking a bit forward I think. :)

**Pull request merged**